content/sensu-go/6.0/reference/filters.md

```diff
@@ -366,6 +366,83 @@ For more information about event attributes, see the [event reference][28].
 `event.entity.system.platform_version` | string  | The entity’s operating system version
 `event.entity.user`                    | string  | Sensu [RBAC][25] username used by the agent entity
 
+## Build event filter expressions with JavaScript execution functions
+
+**COMMERCIAL FEATURE**: Access built-in JavaScript event filter execution functions in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][44].
+
+In addition to [Sensu query expressions][27], Sensu includes several built-in JavaScript functions for event filter execution:
+
+- `sensu.FetchEvent`
+- `sensu.CheckStatus`
+- `sensu.ListEvents`
+
+Use these functions to query your event stores for other events in the same namespace.
+
+For example, to handle only events for the `server01` entity *and* the `disk` check, use the `sensu.FetchEvent` function in your event filter expressions:
+
+{{< code javascript >}}
+"expressions": [
+  '(function () { var diskEvent = sensu.FetchEvent("router01", "disk"); if (diskEvent == nil) { return false; } return diskEvent.check.status == 0; })()'
```

Comment:
You used `server01` as the entity name in the example above, might be better if we stay consistent.